PWGCF/Tasks/correlations.cxx

```diff
@@ -85,6 +87,7 @@ struct CorrelationTask {
 
   O2_DEFINE_CONFIGURABLE(cfgDecayParticleMask, int, 0, "Selection bitmask for the decay particles: 0 = no selection")
   O2_DEFINE_CONFIGURABLE(cfgMassAxis, int, 0, "Use invariant mass axis (0 = OFF, 1 = ON)")
+  O2_DEFINE_CONFIGURABLE(cfgDDBARana, int, 0, "switch for DDBAR analysis (0 = OFF, 1 = ON)");
```

Comment:
Instead of having this, can we check `doprocessSame2Prong2Prong` to enable the additional mass axis and the other features? I assume this would enabled only when that process function is used anyways.